index.html

```diff
@@ -6,25 +6,52 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <title>Awesome Books</title>
+    <script src="https://cdn.jsdelivr.net/npm/luxon@2.0.2/build/global/luxon.min.js"></script>
+    <script src="/js/time.js" defer></script>
     <script src="/index.js" defer></script>
+    <script src="/js/navigation.js" defer></script>
 </head>
 <body>
-    <header></header>
+    <div class="wrapper"></div>
+    <header>
+        <nav class="nav">
+            <span class="nav-title">Awesome Books</span>
+            <div class="nav-items">
+                <a class="list" href="#">List</a>|
+                <a class="add-new" href="#">Add new</a>|
+                <a class="contact" href="#">Contact</a>
+            </div>
+        </nav>
+        <div class="time-date-container">
+            <span class="time-date"></span>
+        </div>
+    </header>
     <main>
         <section class="all-books">
             <h1>All Awesome Books</h1>
             <div class="listShow"></div>
         </section>
-        <hr class="horizontal-line1">
         <section class="add-books">
             <h1>Add A New Book</h1>
-            <div class="input-book">
-                <input type="text" class="title" placeholder="title">
-                <input type="text" class="author" placeholder="author">
+            <form class="input-book">
+                <input type="text" class="title" placeholder="title" required>
+                <input type="text" class="author" placeholder="author" required>
+                <span class="error"></span>
                 <button type="button" class="add">Add</button>
-            </div>
+            </form>
+        </section>
+        <section class="contact-info">
+            <h1>Contact information</h1>
+            <p>Do you have any questions or you just want to say "Hello"?<br>You can reach out to us</p>
+            <ul>
+                <li>Our e-mail mail@mail.com</li>
+                <li>Our phone number: 00421987564123</li>
+                <li>Our address: Streetname 06,19921 City, Country</li>
+            </ul>
         </section>
     </main>
-    <footer></footer>
+    <footer class="copyright">
+        <span class="copyright-text">Copyright &copy; 2021 henry-kc, aliabbani</span>
+    </footer>
```

Comment:
- [ ] As per requirements, the footer should be settling down to the bottom of the page. But right now in our project footer is in the floating state. Kindly try to fix this so the position of the footer should be the same for all navigation links.

Reply:
Alright, thank you